state/batch.go

```diff
@@ -525,3 +540,27 @@ func (s *State) GetLastBatch(ctx context.Context, dbTx pgx.Tx) (*Batch, error) {
 	}
 	return batches[0], nil
 }
+
+// GetBatchTimestamp returns the batch timestamp.
+//
+//	   for >= etrog is stored on virtual_batch.batch_timestamp
+//		  previous batches is stored on batch.timestamp
+func (s *State) GetBatchTimestamp(ctx context.Context, batchNumber uint64, dbTx pgx.Tx) (*BatchTimestampData, error) {
+	forkid := s.GetForkIDByBatchNumber(batchNumber)
+	batchTimestamp, virtualTimestamp, err := s.GetRawBatchTimestamps(ctx, batchNumber, dbTx)
+	if err != nil {
+		return nil, err
+	}
+	if forkid >= FORKID_ETROG {
+		return &BatchTimestampData{
+			BatchTimestamp:   virtualTimestamp,
+			LimitTimestampV2: batchTimestamp,
+			IsV2:             true,
```

Comment:
Why do we need the isV2 param??. If LimitTimestampV2 is nil we are in v1 and if not, we are in v2. I think we could delete it